packages/schema/src/model.ts

```diff
@@ -16,25 +16,69 @@ import {
   IJupyterCadModel,
   ISelection,
   IUserData,
-  Pointer
+  Pointer,
+  IJCadSettings
 } from './interfaces';
 import jcadSchema from './schema/jcad.json';
 import { Contents } from '@jupyterlab/services';
+import { ISettingRegistry } from '@jupyterlab/settingregistry';
+
+const SETTINGS_ID = '@jupytercad/jupytercad-core:jupytercad-settings';
 
 export class JupyterCadModel implements IJupyterCadModel {
   constructor(options: JupyterCadModel.IOptions) {
-    const { annotationModel, sharedModel } = options;
+    const { annotationModel, sharedModel, settingRegistry } = options;
     if (sharedModel) {
       this._sharedModel = sharedModel;
     } else {
       this._sharedModel = this.createSharedModel();
     }
     this._connectSignal();
     this.annotationModel = annotationModel;
+    this.settingRegistry = settingRegistry;
     this._copiedObject = null;
     this._pathChanged = new Signal<JupyterCadModel, string>(this);
   }
 
+  /**
+   * Initialize custom settings for JupyterLab.
+   */
+  readonly settingsChanged = new Signal<this, void>(this);
+
+  async initSettings() {
+    if (this.settingRegistry) {
+      const setting = await this.settingRegistry.load(SETTINGS_ID);
+      this._settings = setting;
+
+      setting.changed.connect(() => {
+        this.settingsChanged.emit();
+        this._updateLocalSettings();
+      });
+    }
+  }
+
+  private _updateLocalSettings(): void {
+    const composite = this._settings.composite;
+
+    this._jcadSettings = {
+      showAxesHelper: (composite.showAxesHelper as boolean) ?? false,
+      cameraType:
+        (composite.cameraType as 'Perspective' | 'Orthographic') ??
+        'Perspective'
+    };
+  }
+
+  jcadSettings(): IJCadSettings {
```

Comment:
why not use a getter?

Reply:
Done in [529e101](https://github.com/jupytercad/JupyterCAD/pull/725/commits/529e10117a135c8138b2eb255237121f61fe9eef)